cmd/gossamer/configcmd.go

```diff
@@ -328,3 +338,24 @@ var tomlSettings = toml.Config{
 		return fmt.Errorf("field '%s' is not defined in %s%s", field, rt.String(), link)
 	},
 }
+
+// tilde will expand a tilde prefix path to full home path
+func tilde(targetPath string) string {
+	if strings.HasPrefix(targetPath, "~\\") || strings.HasPrefix(targetPath, "~/") {
+		if homeDir := home(); homeDir != "" {
+			targetPath = homeDir + targetPath[1:]
+		}
+	}
+	return path.Clean(os.ExpandEnv(targetPath))
+}
+
+// home will return the HOME ENV DIR OR the current user dir
+func home() string {
+	if homeDir := os.Getenv("HOME"); homeDir != "" {
+		return homeDir
+	}
+	if usrDir, err := user.Current(); err == nil {
+		return usrDir.HomeDir
+	}
+	return ""
+}
```

Comment:
Same as `cfg.homeDir()`? Maybe it makes sense for this logic to live there, up to you

Reply:
`defaultConfig` maybe not the best name here, but its the name of the struct that is being marshaled into, but `fig` as a var name was also not good either. Do you have a better suggestion ?

Reply:
Is this resolved by the comment above?

Reply:
I agree, and keeping this consistent with the name used in `configcmd.go` make sense